app/vmui/packages/vmui/src/components/ExploreMetrics/ExploreMetricsHeader/ExploreMetricsHeader.tsx

```diff
@@ -38,59 +44,99 @@ const ExploreMetricsHeader: FC<ExploreMetricsHeaderProps> = ({
-          label="Instance"
-          placeholder="Please select instance"
-          onChange={onChangeInstance}
-          noOptionsText={noInstanceText}
-          clearable
-          searchable
-        />
+    <>
+      <div
+        className={classNames({
+          "vm-explore-metrics-header": true,
+          "vm-explore-metrics-header_mobile": isMobile,
+          "vm-block": true,
+          "vm-block_mobile": isMobile,
+        })}
+      >
+        <div className="vm-explore-metrics-header__job">
+          <Select
+            value={job}
+            list={jobs}
+            label="Job"
+            placeholder="Please select job"
+            onChange={onChangeJob}
+            autofocus={!job}
+            searchable
+          />
+        </div>
+        <div className="vm-explore-metrics-header__instance">
+          <Select
+            value={instance}
+            list={instances}
+            label="Instance"
+            placeholder="Please select instance"
+            onChange={onChangeInstance}
+            noOptionsText={noInstanceText}
+            clearable
+            searchable
+          />
+        </div>
+        <div className="vm-explore-metrics-header__size">
+          <Select
+            label="Size graphs"
+            value={size}
+            list={sizeOptions}
+            onChange={onChangeSize}
+          />
+          <Tooltip title={`${showTips ? "Hide" : "Show"} tip`}>
+            <Button
+              variant="text"
+              color={showTips ? "warning" : "gray"}
+              startIcon={<TipIcon/>}
+              onClick={toggleShowTips}
+            />
+          </Tooltip>
+        </div>
+        <div className="vm-explore-metrics-header-metrics">
+          <Select
+            label={"Metrics"}
+            value={selectedMetrics}
+            list={names}
+            placeholder="Search metric name"
+            onChange={onToggleMetric}
+            noOptionsText={noMetricsText}
+            clearable
+            searchable
+          />
+        </div>
       </div>
-      <div className="vm-explore-metrics-header__size">
-        <Select
-          label="Size graphs"
-          value={size}
-          list={sizeOptions}
-          onChange={onChangeSize}
-        />
-      </div>
-      <div className="vm-explore-metrics-header-metrics">
-        <Select
-          label={"Metrics"}
-          value={selectedMetrics}
-          list={names}
-          placeholder="Search metric name"
-          onChange={onToggleMetric}
-          noOptionsText={noMetricsText}
-          clearable
-          searchable
-        />
-      </div>
-    </div>
+
+      {showTips && (
+        <Alert variant={"warning"}>
+          <div className="vm-explore-metrics-header-description">
+            <p>
+              Please note: this page is solely designed for exploring Prometheus metrics.
+              Prometheus metrics always carry <code>job</code> and <code>instance</code> labels,
```

Comment:
```suggestion
              Prometheus metrics always contain <code>job</code> and <code>instance</code> labels,
```

Reply:
Would be nice to link the convention here https://prometheus.io/docs/concepts/jobs_instances/